pkg/storage/chunk/cache/fifo_cache.go

```diff
@@ -32,18 +32,22 @@ const (
 type FifoCacheConfig struct {
 	MaxSizeBytes string        `yaml:"max_size_bytes"`
 	MaxSizeItems int           `yaml:"max_size_items"`
-	Validity     time.Duration `yaml:"validity"`
```

Comment:
I think we have to deprecate it before deleting it.

This is because people having the value will not get their loki start.

Reply:
ho nevermind I didn't realize you renamed also that one.